docs/references/protocol/transactions/transaction-results/ter-codes.md

```diff
@@ -10,18 +10,20 @@ labels:
 
-**Note:** Transactions with `ter` codes have not been applied to the current ledger and have not yet changed the XRP Ledger state. A transaction that provisionally got a `ter` result may still succeed or fail with a different code after being automatically applied later. For more information, see [Finality of Results](../../../../concepts/transactions/finality-of-results/index.md) and [Reliable Transaction Submission](../../../../concepts/transactions/reliable-transaction-submission.md).
+{% admonition type="info" name="Note" %}
+Transactions with `ter` codes have not been applied to the current ledger and have not yet changed the XRP Ledger state. A transaction that provisionally got a `ter` result may still succeed or fail with a different code after being automatically applied later. For more information, see [Finality of Results](../../../../concepts/transactions/finality-of-results/index.md) and [Reliable Transaction Submission](../../../../concepts/transactions/reliable-transaction-submission.md).
+{% /admonition %}
 
 | Code             | Explanation                                               |
 |:-----------------|:----------------------------------------------------------|
 | `terFUNDS_SPENT` | **DEPRECATED.**                                           |
 | `terINSUF_FEE_B` | The account sending the transaction does not have enough XRP to pay the `Fee` specified in the transaction. |
 | `terLAST`        | Used internally only. This code should never be returned. |
 | `terNO_ACCOUNT`  | The address sending the transaction is not funded in the ledger (yet). |
-| `terNO_AMM`      | The AMM-related transaction specifies an asset pair that does not currently have an AMM instance. _(Requires the [AMM amendment][])_ |
+| `terNO_AMM`      | The AMM-related transaction specifies an asset pair that does not currently have an AMM instance. _(Added by the [AMM amendment][])_ |
 | `terNO_AUTH`     | The transaction would involve adding currency issued by an account with `lsfRequireAuth` enabled to a trust line that is not authorized. For example, you placed an offer to buy a currency you aren't authorized to hold. |
 | `terNO_LINE`     | Used internally only. This code should never be returned. |
-| `terNO_RIPPLE`   | Used internally only. This code should never be returned. |
+| `terNO_RIPPLE`   | The transaction can't succeed because of No Ripple settings on trust lines. For example, the Check tried to send a token payment that would be blocked because of No Ripple flags. Alternatively, the transaction tried to create an AMM even though the issuer of at least one of the tokens has not enabled the [Default Ripple flag](/docs/concepts/tokens/fungible-tokens/rippling/#the-default-ripple-flag). |
```

Comment:
Should the phrase "No Ripple" be linked somewhere or in code mode?

Reply:
Not code font; because flags have multiple different names in code (e.g. `lsf` vs `tf` prefix) we use prose names like "Default Ripple" and "No Ripple" that don't need code font. As for the link it would go to the same page as the Default Ripple one below, so I think it makes the most sense to link that page earlier in the description (and not re-link parts of it later in the same line).